modules/openapi-generator-core/src/main/java/org/openapitools/codegen/config/WorkflowSettings.java

```diff
@@ -362,6 +375,17 @@ public Builder withRemoveOperationIdPrefix(Boolean removeOperationIdPrefix) {
             return this;
         }
 
+        /**
+         * Sets the {@code skipOperationExample} and returns a reference to this Builder so that the methods can be chained together.
+         *
+         * @param skipOperationExample the {@code skipOperationExample} to set
+         * @return a reference to this Builder
+         */
+        public Builder withSkipOperationExample(Boolean skipOperationExample) {
+            this.skipOperationExample = skipOperationExample != null ? skipOperationExample : Boolean.valueOf(DEFAULT_REMOVE_OPERATION_ID_PREFIX);
```

Comment:
This appears to be the wrong default that is assigned - should be DEFAULT_SKIP_OPERATION_EXAMPLE i suppose